extensions/amp-install-serviceworker/0.1/test/test-amp-install-serviceworker.js

```diff
@@ -206,23 +208,35 @@ describes.realWin('amp-install-serviceworker', {
       const iframeSrc = 'https://www.example.com/install-sw.html';
       install.setAttribute('data-iframe-src', iframeSrc);
       implementation.buildCallback();
+      let iframe;
       const appendChild = install.appendChild;
       install.appendChild = child => {
-        child.complete = true; // Mark as loaded.
-        expect(child.src).to.equal(iframeSrc);
-        child.src = 'about:blank';
-        appendChild.call(install, child);
+        iframe = child;
+        iframe.complete = true; // Mark as loaded.
+        expect(iframe.src).to.equal(iframeSrc);
+        iframe.src = 'about:blank';
+        appendChild.call(install, iframe);
       };
-      expect(install.children).to.have.lengthOf(0);
-      implementation.layoutCallback();
-      expect(install.children).to.have.lengthOf(1);
-      const iframe = install.children[0];
-      expect(iframe).to.exist;
-      expect(calledSrc).to.undefined;
-      expect(install.style.display).to.equal('none');
-      expect(iframe.tagName).to.equal('IFRAME');
-      expect(iframe.getAttribute('sandbox')).to.equal(
-          'allow-same-origin allow-scripts');
+      let deferredMutate;
+      implementation.deferMutate = fn => {
```

Comment:
Same as above: https://github.com/ampproject/amphtml/commit/c4ef8697f5084ecbeef85d47317fc3831a643010#diff-c7e92f692652c6ad5db6167bf7608b8e

Reply:
Also, you can move anything relating to LayoutPriority because it's no longer part of this extension